cpp-ch/local-engine/local_engine_jni.cpp

```diff
@@ -166,6 +167,8 @@ JNIEXPORT jint JNI_OnLoad(JavaVM * vm, void * /*reserved*/)
     local_engine::BroadCastJoinBuilder::init(env);
     local_engine::CacheManager::initJNI(env);
 
+    DB::registerFormats();
```

Comment:
Why does this PR need to call `DB:: tradeFormats()`?

Reply:
dump block needs formats.

Reply:
adding a simple method to dump block contents is very useful for debug

Reply:
You mean calling `DB:: registerFormats()` is for later dumping?